### DISCUSSION TITLE
ESDL format conversion

### DISCUSSION THREAD

**Discussion by jvandorstrhdhv** [Project-OMOTES/user_feedback#65] [Q&A]
With our own tooling we generate distribution networks. We want to convert those distribution networks to ESDL format so we can import those into the Design Toolkit. For this we want to know what information is required for the conversion to ESDL format so that it can be used in the Design Toolkit.

We would prefer to discuss this during a short meeting so we can brainstorm.

**Comment by ewoudwerkman** [comment]
Let's discuss this in this Thursday's session, to give you some pointers on converting existing data into ESDL.

**Reply by jvandorstrhdhv** [reply]
unfortunately I cannot attend the session thursday because I am on holiday. It is possible to move this to the session November 21th?

**Reply by ewoudwerkman** [reply]
Fine with me! @jarsarasty Ok with you?

**Reply by jarsarasty** [reply]
Thanks @ewoudwerkman! 
Yes, November 21st works for me.

**Reply by jvandorstrhdhv** [reply]
Hi, will this subject still be discussed the 21st now the partner support format has changed and is moved to 14:00?

**Reply by jarsarasty** [reply]
Hi @jvandorstrhdhv

Yes, this discussion is included in the agenda for Thursday next week at 14:00.

**Comment by jarsarasty** [comment]
Python implementation of the Energy System Description Language (ESDL) for modelling energy systems: [https://pypi.org/project/pyESDL/](https://pypi.org/project/pyESDL/)

**Comment by jarsarasty** [comment]
PyESDL Documentation: [https://pyesdl.readthedocs.io/en/latest/](https://pyesdl.readthedocs.io/en/latest/)

**Comment by jarsarasty** [comment]
Heat Network Shapefile to ESDL conversion: [https://github.com/EnergyTransition/shapefile-2-esdl](https://github.com/EnergyTransition/shapefile-2-esdl)

**Comment by jarsarasty** [comment]
Create ESDL from Excel: [https://github.com/EnergyTransition/excel-2-esdl](https://github.com/EnergyTransition/excel-2-esdl)

**Comment by jarsarasty** [comment]
ESDL documentation (a bit outdated): [https://energytransition.gitbook.io/esdl](https://energytransition.gitbook.io/esdl)

**Comment by edwinmatthijssen** [comment]
ESDL website: https://www.esdl.nl/

**Comment by jarsarasty** [comment]
ESDL Model Reference documentation [https://energytransition.github.io/index.html](https://energytransition.github.io/index.html)
